pages/linux/guake.md

```diff
@@ -0,0 +1,27 @@
+# guake
+
+> Drop-down terminal for GNOME.
+
+- Toggle Guake Visiblity:
+
+`F12`
+
+- Toggle Fullscreen:
+
+`F11`
+
+- Open New Tab:
```

Comment:
```suggestion
- Open a new tab:
```

Reply:
Thanks for the advice, it does read much better. :thumbsup: